### DISCUSSION TITLE
Doris 大小写问题

### DISCUSSION THREAD

**Discussion by gilboom** [apache/doris#24908] [RFC]
请问为什么 Doris 里面的数据在查询的时候是大小写敏感的呢？使用 `show collation` 命令，看到唯一支持的 collation 是 `utf8_general_ci`，`ci` 不应该是大小写不敏感的么？
![image](https://github.com/apache/doris/assets/23697209/cc3a7521-7419-42fa-8916-2667821f25dd)

有没有办法实现大小写不敏感呢？

**Comment by LemonLiTree** [comment]
try use lower() function

**Reply by gilboom** [reply]
It works for me，Thank you 😆

**Comment by syhxj** [comment]
like '%keyword%'  =>  regexp '(?i)keyword'
